scripts/bnsd/genesis_app_state.json

```diff
@@ -165,6 +191,14 @@
       "owner": "cond:gov/rule/0000000000000002",
       "valid_username_name": "^[a-z0-9\\-_.]{3,64}$",
       "valid_username_label": "^iov$"
+    },
+    "account": {
+      "valid_name": "^[a-z0-9\\-_.]{3,64}$",
+      "valid_domain": "^[a-z0-9\\-_]+${3,16}",
```

Comment:
Missing `$` at end of regex.

Reply:
Great catch.  The regex should be `^[a-z0-9\\-_]{3,16}$`.